3.process_cfret_features/scripts/1.single_cell_processing.py

```diff
@@ -95,14 +95,23 @@
 
     # Save the modified DataFrame back to the same location
     annotated_df.to_parquet(output_annotated_file, index=False)
+    
+    # Only for Plate 4, we want to normalize to the DMSO treatments
+    if plate == "localhost231120090001":
+        samples = "Metadata_heart_number == 7 and Metadata_treatment == 'DMSO'"
+    # For all other plates, use the default
+    else:
+        samples = "all"
```

Comment:
```suggestion
    # for all plates
     samples = "all"
     
    # Only for Plate 4, we want to normalize to the DMSO treatments
    if plate == "localhost231120090001":
        samples = "Metadata_heart_number == 7 and Metadata_treatment == 'DMSO'"
```
Some suggestion of making the code a lot cleaner and avoiding else clauses

Reply:
Love this! Just implemented!